src/ducks/balance/SetFilterAndRedirect.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect } from 'react'
+import { useDispatch } from 'react-redux'
+import { useQuery } from 'cozy-client'
+import { useRouter, useParams } from 'components/RouterContext'
+import Loading from 'components/Loading'
+import {
+  accountsConn,
+  groupsConn,
+  ACCOUNT_DOCTYPE,
+  GROUP_DOCTYPE
+} from 'doctypes'
+import { filterByDoc } from '.'
+
+/**
+ * - Checks the params from the URL
+ * - Sets filter doc accordingly
+ * - Redirects to the right page
+ */
+const mkSetFilterAndRedirect = () =>
+  function SetFilterAndRedirect() {
+    const router = useRouter()
+    const params = useParams()
+    const dispatch = useDispatch()
+    const accounts = useQuery(accountsConn.query, accountsConn)
+    const groups = useQuery(groupsConn.query, groupsConn)
+    useEffect(() => {
+      if (
+        !accounts.fetchStatus === 'loading' ||
```

Comment:
I have an hard time to understand this condition 🤔

Reply:
It was a typo. Anyway, after talking to Flo, we concluded that we did not want to keep the condition, since we have push notifications that could be sent without being configured in
settings (for example konnector alerts).